src/main/webapp/app/admin/metrics/metrics.component.css

```diff
@@ -11,12 +11,22 @@ Metrics page styles
     text-align: initial;
 }
 
-span.label { /* UP, DOWN, ... */
+/* ==========================================================================
+Routes styles
```

Comment:
why is this called route styles?
Please move this into the main app css and make it  class and apply it everywhere instead of duplicating this everywhere